packages/aws_iam_privesc/PKGBUILD

```
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=aws_iam_privesc
```

Maybe

```suggestion
pkgname=aws-iam-privesc
_pkgname=aws_iam_privesc
```

and replace `$pkgname` with `$_pkgname`